pkg/fanal/analyzer/language/dotnet/deps/deps_test.go

```diff
@@ -21,12 +21,12 @@ func Test_depsLibraryAnalyzer_Analyze(t *testing.T) {
 	}{
 		{
 			name:      "happy path",
-			inputFile: "testdata/datacollector.deps.json",
+			inputFile: "../../../../../../integration/testdata/fanal/dotnet/deps/datacollector.deps.json",
```

Comment:
Do you know if we can specify a relative path from the project root? Like `./integration/testdata/`

Reply:
unfortunately, i don't know how to use a relative path, `./integration/testdata...` doesn't work.
I can take a look at workaround...

also I've seen such paths here:
https://github.com/aquasecurity/trivy/blob/f8dd616545844fc244855a839268b3315180f3e8/pkg/fanal/test/integration/library_test.go#L49

Reply:
Yes, I didn't like it, but have no time to look for a better way at that time.

Reply:
I tried to use something like `os.Getwd()` - it doesn't work too. I can try to figure out a better way